TelegramTest/AppDelegate.m

```diff
@@ -738,14 +741,7 @@ typedef NS_OPTIONS(NSUInteger, NSEventModifierFlags) {
                 [TMViewController becomePasslock];
             }
         }
-        
-        if(result.keyCode == 48) {
-          //  NSTextView *textView = responder;
-            
-            
-            return [[NSEvent alloc]init];
-        }
-        
+
```

Comment:
I'm not sure why this chunk is here but it blocks the tab events from propagating.